plugins/worldinterface/src/worldproxy.h

```diff
@@ -0,0 +1,193 @@
+// -*- mode:C++; tab-width:4; c-basic-offset:4; indent-tabs-mode:nil -*-
+/*
+ * Copyright (C) 2015 iCub Facility 
+ * Authors: Lorenzo Natale
+ * CopyPolicy: Released under the terms of the LGPLv2.1 or later, see LGPL.TXT
+ *
+ */
+
+#ifndef __WORLDPROXY__
+#define __WORLDPROXY__
```

Comment:
A modification just for being picky: the C++ standard reserves the use of identifier starting with a double underscore:
~~~
— Each identifier that contains a double underscore __ or begins with an underscore followed by an uppercase letter is reserved to the implementation for any use.
— Each identifier that begins with an underscore is reserved to the implementation for use as a name in the global namespace.
~~~
I'll modify it to fit what we use in the other files.